README.md

```diff
@@ -205,3 +205,17 @@ This phase is were we convert the documentation to code and make everything work
 		</tr>
 	</tbody>
 </table>
+
+### SetUp
+
+1. Run <code>yarn start:dev</code>
+2. Run econominhas-api migrations
+   - Step 2.1: Run docker ps for get id container <code>econominhas-api</code>.
+   - Step 2.2: Run docker exec -it <code>\<ID container\></code>.
+   - Step 2.3: Run <code>yarn db:migrate</code> to create the migrations.
```

Comment:
Please, add a step to change the `.env` from:
```
DATABASE_URL=postgresql://username:password@postgres:5432/database?schema=public
```
to
```
DATABASE_URL=postgresql://username:password@localhost:5432/database?schema=public
```

And another one at the end to revert this change.